addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1622,6 +1648,27 @@ MockServer.include({
         ]);
         this._mockUnlink(model, [followers.map(follower => follower.id)]);
     },
+    /**
+     * Simulates `set_mail_user_settings` on `mail.user.settings`.
+     *
+     * @param {integer} id
+     * @param {object.<string, boolean>} new_settings
```

Comment:
just Object is enough, it is not guaranteed that all settings will be boolean (it actually won't be the case in rtc branch)

Reply:
done